packages/design-mui/src/components/MuiProvider/MuiProvider.tsx

```diff
@@ -3,41 +3,24 @@ import type { XProviderProps } from "@hatchifyjs/react-ui"
 import {
   HatchifyPresentationProvider,
   HatchifyPresentationDefaultValueComponents,
-  HatchifyPresentationDefaultFieldComponents,
 } from "@hatchifyjs/react-ui"
 import { MuiLayout } from "../MuiLayout"
-import { MuiForm } from "../MuiForm"
-import { MuiDetails } from "../MuiDetails"
 import MuiDataGrid from "../MuiDataGrid/MuiDataGrid"
-import {
-  String as StringInput,
-  Number as NumberInput,
-  Date as DateInput,
-  Boolean as BooleanInput,
-  Relationship as RelationshipInput,
-} from "./DefaultFieldComponents"
 import { Relationship, RelationshipList } from "./DefaultDisplayComponents"
 
 export const MuiProvider: React.FC<XProviderProps<Theme>> = ({ children }) => {
   return (
     <HatchifyPresentationProvider
       Collection={MuiDataGrid}
       Layout={MuiLayout}
-      Details={MuiDetails}
-      Form={MuiForm}
+      // Details
+      // Form
       defaultValueComponents={{
         ...HatchifyPresentationDefaultValueComponents,
         Relationship,
         RelationshipList,
       }}
-      defaultFieldComponents={{
-        ...HatchifyPresentationDefaultFieldComponents,
-        String: StringInput,
-        Date: DateInput,
-        Number: NumberInput,
-        Boolean: BooleanInput,
-        Relationship: RelationshipInput,
-      }}
+      // defaultFieldComponents
```

Comment:
Same comment as above

Reply:
ok i've decided to keep these but prefix the comments with `future:`, otherwise it looks like you can comment it out and expect it to work lol